makefiles/vars.inc.mk

```diff
@@ -15,6 +15,7 @@ export CPU_MODEL             # The specific identifier of the used CPU, used for
 export CPU_CORE              # The specific identifier of the core present in the CPU. Needed for depency resolution.
 export CPU_ARCH              # The specific identifier of the architecture of the core defined in CPU_CORE.
 export CPU_FAM               # An intermediate identifier between CPU and CPU_MODEL that represents a sub-group of a Manufacturers CPU's.
+export CPU_LINE              # An intermediate identifier between CPU_FAM and CPU_MODEL that represents a sub-group of a STM32 CPU's
```

Comment:
Is this needed?

Reply:
I could remove it from a global export here but still have to export it when building the vectors .o file.

Reply:
why?

Reply:
Because the CPU_LINE variable is required to use the right file when building the vectors module. But I could export it only to build this object and not globally like before.